grpc-tls-quickstart/README.md

```diff
@@ -27,7 +27,7 @@ The project has gRPC server reflection enabled (see `application.properties`), s
 the gRPC service directly with [grpcurl](https://github.com/fullstorydev/grpcurl).
 
 ```
-grpcurl --cacert src/main/resources/tls/ca.pem -d '{"name": "World!"}' localhost:9000 helloworld.Greeter/SayHello
+grpcurl --cacert src/main/resources/META-INF/resources/tls/ca.pem -d '{"name": "World!"}' localhost:9000 helloworld.Greeter/SayHello
```

Comment:
I don't think that's a good idea to do that. The `META-INF/resources` directory is publicly available.

Reply:
I've used `META-INF/resources` because it's used on other quickstarts:
[security-jwt-quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/master/security-jwt-quickstart)
[security-oauth2-quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/master/security-oauth2-quickstart)

Do you have a suggestion for an alternative directory ?

Thanks

Reply:
So, in fact, it's in this directory because the files are packaged with the native executable. I don't believe that it's a good idea, it needs to be somewhere else and the extension should add them to the native executable.